common/common-test-setup/src/base_setup/checks.rs

```diff
@@ -142,7 +142,24 @@ impl BaseSetup {
                     .collect();
                 for token in tokens {
                     let (token_id, amount) = token;
-                    assert!(sc.deposited_tokens_amount(&token_id).get() == amount);
+                    let deposited_token_handle = sc.deposited_tokens_amount(&token_id);
```

Comment:
This looks unnecessary. Can just use the previous assert with some error message